src/core/migrations/0125_auto_20160505_1133.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+from core import models as core_models
+
+def load_setting(apps, schema_editor):
+    # We can't import the Person model directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    setting_group, created = core_models.SettingGroup.objects.get_or_create(name='email', enabled=True)
```

Comment:
Also, I've found it useful to rename auto-generated migrations. You could name this `0125_add_change_principal_contact_proposal_setting.py`.